### PR TITLE
Revert classname

### PR DESCRIPTION
**Pull request by sutser**

### CODE REVIEW

File: zxcvbn-test/ZxcvbnTest.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public class ZxcvbnTest
         [TestMethod]
         public void RunAllTestPasswords()
         {
-            var zx = new Zxcvbn.Zxcvbn(new Zxcvbn.DefaultMatcherFactory());
+            var zx = new Zxcvbn.PasswordMetric(new Zxcvbn.DefaultMatcherFactory());
```
Comment (dngroth) [inline zxcvbn-test/ZxcvbnTest.cs]: Is the namespace `Zxcvbn` necessary?

File: zxcvbn-cs/README.md [hunk 2]
```diff
@@ -101,3 +101,7 @@ The following changes differ from the [original library](https://github.com/mick
 	
 - Change dictionaries load
 	Loads all embedded resources from the directory "zxcvbn.Dictionaries"
```
Comment (dngroth) [inline zxcvbn-cs/README.md:103]: Add translation changes

**Review by dngroth — CHANGES_REQUESTED** [review]

File: zxcvbn-test/ZxcvbnTest.cs (hunk 1 above)
Comment (sutser) [reply zxcvbn-test/ZxcvbnTest.cs]: Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/5a055b5c16c4173c813dadf7d1b0f179903377ca

Comment (sutser) [reply zxcvbn-test/ZxcvbnTest.cs]: Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/ad996c0914ef81f3d118fb758225b73a8e0e2213

File: zxcvbn-cs/README.md (hunk 2 above)
Comment (sutser) [reply zxcvbn-cs/README.md:103]: Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/13379cfef9e4d4413c842d1c21901feaddfbcc36

Comment (sutser) [reply zxcvbn-cs/README.md:103]: Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/5627d9ebe657a3afc2b05c36bce9bd63a9f2e7b6

File: zxcvbn-cs/README.md [hunk 3]
```diff
@@ -101,3 +101,17 @@ The following changes differ from the [original library](https://github.com/mick
...
+
+- Rename class "Zxcvbn" to "PasswordMetric"
+
+- Rename class "Result" to "PasswordMetricResult"
+
+- Rename translation value "France" to "French"
+
+- Add translation value "Italian"
+
+- Remove not needed namespaces
```
Comment (dngroth) [inline zxcvbn-cs/README.md]: This is not an api change

**Review by dngroth — CHANGES_REQUESTED** [review]

File: zxcvbn-cs/README.md (hunk 3 above)
Comment (sutser) [reply zxcvbn-cs/README.md]: Addressed in: https://github.com/messerli-informatik-ag/zxcvbn-cs/pull/4/commits/95d63aaa67f97ae5e0c0dea5278a314159845cb6

**Review by dngroth — APPROVED** [review]

**Pull request merged**